earth_enterprise/src/server/geresetpgdb

```diff
@@ -201,27 +207,49 @@ if [ -e $VARPGDIR/data/postmaster.pid ]; then
 fi
 
 if [ $type == "backup" ]; then
-  if [ ! -e $VARPGDIR/data ]; then
-    echo "WARNING: There is no data to back up."
-  else
-    do_dump
-  fi
-  exit
+  do_dump
 fi
 
 # if the data dir does not exist then this has to be a hard reset.
+# TODO: This will need to be removed when we unbundle
```

Comment:
minor: unbundle **PostgreSQL**
so it a bit more clear to any one else not familiar with effort to unbundle ... ?